storybooknative/src/Touchable.js

```diff
@@ -51,6 +53,11 @@ export default class Touchable extends React.Component<Props> {
     // but since we are building for both platforms it is here as well
     // to discover this mistake as soon as possible
     const children = React.Children.only(this.props.children);
+    const styles = StyleSheet.create({
```

Comment:
Also, I think we should stick to styled components to handle the styling.

Reply:
Absolutely, good catch